pkg/kubectl/cmd/create.go

```diff
@@ -53,12 +54,13 @@ func NewCmdCreate(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 	options := &CreateOptions{}
 
 	cmd := &cobra.Command{
-		Use:     "create -f FILENAME",
+		Use:     "create [ -f FILENAME | -F FILES-FROM ]",
```

Comment:
`create (-f FILENAME | --files-from=FILENAME)` 

either one or another is required so `()` instead of `[]`